psi4/src/psi4/fnocc/df_scs.cc

```diff
@@ -90,22 +90,73 @@ std::tuple<double, double, SharedMatrix, SharedMatrix> DFCoupledCluster::Compute
             ssenergy += pair_ss;
             matAA->add(i, j, pair_ss);
             matAB->add(i, j, pair_os);
+            if (i != j) {
+                matAA->add(j, i, pair_ss);
+                matAB->add(j, i, pair_os);
+            }
```

Comment:
The equation this code uses is e_ij = \sum_ab [ 1/2 g^ij_ab c^ab_ij + 1/2 g^ji_ab c^ab_ji ].

For a given ij pair, these two contributions are not the same. The C amplitude may only differ by a sign, but because the integrals are not antisymmetrized, you can think of these as a separate Coulomb and Exchange contribution. When we process (i, j), we add the Coulomb-like part to the (i, j) pair the Exchange-like part to the (j, i) pair.

The equations are a little more complicated when you deal with spin-integration, but that's the idea. I can add a comment.

Reply:
A comment would be helpful, thanks. I think I was confused because I temporarily forgot that we're no longer using the convention that the sum of the pair-energy matrix (in this case `matAA` and `matAB`) equals the total energy. Instead, the sum of the lower diagonal (including the diagonal itself) should equal the total energy. Correct me if I'm wrong.